staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -1321,14 +1339,14 @@ func (r *Request) transformResponse(resp *http.Response, req *http.Request) Resu
 }
 
 // truncateBody decides if the body should be truncated, based on the glog Verbosity.
-func truncateBody(body string) string {
+func truncateBody(logger klog.Logger, body string) string {
```

Comment:
nit: log

Reply:
As parameter name? It's called `logger` elsewhere, including the logr package itself:

https://github.com/go-logr/logr/blob/275154abd02f681c2e10b42ab308c4ca3b16c6be/logr.go#L32-L38

Reply:
Got used too much to `log` in other code bases I guess. Seems to be more Golang-like. But just my taste :)

Reply:
Other code bases uses logr? Just wondering. I think we should follow the example set in logr itself.